vpr/src/base/read_options.cpp

```diff
@@ -2176,6 +2176,15 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .choices({"move_type", "move_block_type"})
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument<bool, ParseOnOff>(args.place_re_cluster, "--place_re_cluster")
+        .help(
+            "Use this option to determine whether reclustering occurs during placement. "
+            ""
+            "When this option is set to 'on,' the placement stage may change some clusters. "
+            "Conversely, if the option is set to 'off,' the clustering determined by the packer will remain unchanged")
```

Comment:
I think it would be a good idea to put the default value in here as well so it is clear what it is by default.

Reply:
It has a default value, and it is set to "off"